src/EventHub.Domain/Organizations/Plans/FeatureOfPlanDefinition.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using JetBrains.Annotations;
+
+namespace EventHub.Organizations.Plans;
+
+public class FeatureOfPlanDefinition
+{
+    [CanBeNull]
+    [Range(0, uint.MaxValue, ErrorMessage = "Value for {0} must be between {1} and {2}.")]
```

Comment:
What about localization? We normally don't set an error message, ABP handles it.